src/cr/cube/cubepart.py

```diff
@@ -775,9 +775,26 @@ def pairwise_significance_tests(self):
 
     @lazyproperty
     def population_counts(self):
-        return (
-            self.table_proportions * self._population * self._cube.population_fraction
+        """2D np.float64 ndarray of population counts per cell.
+
+        The (estimated) population count is computed based on the `population` value
+        provided when the Slice is created. It is also adjusted to account for any
+        filters that were applied as part of the query.
+
+        If any of the dimensions (rows or columns) is a categorical date, the
+        appropriate percentages are used, to calculate the population counts, so as to
+        be the total amount among categorical dates.
+
+        Otherwise, table percents are used to calculate population counts.
+        """
+        proportions = (
+            self.row_proportions
+            if self._rows_dimension.dimension_type == DT.CAT_DATE
```

Comment:
at slice level we have `rows_dimension_type` property ... so you can use that one